kubeflow-pipelines.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: kubeflow-pipelines
-  version: 2.3.0
```

Comment:
Why is this downgrading to 2.0.3?

Anything that depends on `kubeflow-pipelines` without selecting a version will pick up the latest package, which is 2.3.0 and won't include your changes here.

Reply:
Oh I see 2.0.3 looks like the latest version, but KFP SDK is 2.4.0 with a previous 2.3.0: https://github.com/kubeflow/pipelines/releases

Do we need to withdraw some packages?

Reply:
the latest version is https://github.com/kubeflow/pipelines/releases/tag/2.0.3. they also use sdk- version to keep track kfp sdk's only.

Reply:
I noticed that when I looked at the tags of the images of the kubeflow-pipelines components. The latest is 2.0.3

Reply:
> Do we need to withdraw some packages?
I think so yes!

Reply:
added packages that needs to be withdrawn to withdrawn.txt.